src/main/frontend/components/block.cljs

```diff
@@ -546,7 +541,9 @@
                (str " page-property-key block-property")
                untitled? (str " opacity-50"))
       :data-ref page-name
-      :on-mouse-down (fn [e] (open-page-ref e page-name redirect-page-name page-name-in-block contents-page? whiteboard-page?))
+      :draggable true
+      :on-drag-start (fn [e] (editor-handler/block->data-transfer! page-name e))
+      :on-mouse-up (fn [e] (open-page-ref e page-name redirect-page-name page-name-in-block contents-page? whiteboard-page?))
```

Comment:
Using mouse-up instead of mouse-down on refs improves the overall experience and allows dragging and dropping refs to the canvas. This behavior can be enhanced in the future, if we want to allow dropping refs to the outliner.